editor/scripts/bundle.py

```diff
@@ -146,19 +146,36 @@ def mac_certificate(codesigning_identity):
     else:
         return None
 
-def sign_files(codesigning_identity, bundle_dir):
-    certificate = mac_certificate(codesigning_identity)
-    if certificate == None:
-        print("Warning: Codesigning certificate not found for signing identity %s, files will not be signed" % (codesigning_identity))
-    else:
-        exec_command([
-            'codesign',
-            '--deep',
-            '--force',
-            '--options', 'runtime',
-            '--entitlements', './scripts/entitlements.plist',
-            '-s', certificate,
-            bundle_dir])
+def sign_files(platform, options, dir):
+    if 'win32' in platform:
+        certificate = options.windows_cert
+        certificate_pass = options.windows_cert_pass
+        if certificate == None:
+            print("Warning: Codesigning certificate not found, files will not be signed")
```

Comment:
Is it really just a warning?
We called "sign_files", but it only warns, and it doesn't return an error, so any CI step will be green.

Reply:
I'm just thinking that if you bundle the editor locally you don't want an error. Although we should maybe check options.windows_cert and not call sign_files() if it is None

Reply:
I added a --skip-codesign flag for the editor bundle.py script. If this flag isn't set you'll now get an error when you try to bundle and the certificate or code signing identity is missing